docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -98,7 +98,7 @@ We used a [`jetson` board](/components/board/jetson/), but you can use any model
 ![Configuration of a jetson board with digital interrupts in the Viam app config builder.](/tutorials/navigate-with-rover-base/board-config-builder.png)
 
 2. Configure [digital interrupts](/components/board/#digital_interrupts) on your board to signal precise GPIO state changes to the [encoders](/components/encoder/) on your rover base.
-   Copy and paste the following into your board's **Attributes** to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
+   Copy and paste the following into your board's attributes box to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
```

Comment:
Box isn't the correct term. It's a textarea I think but no one says that, field works I think?
```suggestion
   Copy and paste the following into your board's **Attributes** field to add [digital interrupts](/components/board/#digital_interrupts) on pins `31`, `29`, `23`, and `21`:
```